push/sender/pom.xml

```diff
@@ -112,6 +112,12 @@
             <version>0.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.aerogear</groupId>
+            <artifactId>java-adm</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```

Comment:
let's get that guy to maven central and fix the outstanding review issues. 
For version shipped to maven central, let's not use 0.0.1 :-)